### PR TITLE
Release 1.0

### PR DESCRIPTION
**Pull request by alena0817**
further refactoring: encapsulate route_handling.js logic into one  object

### CODE REVIEW

File: lib/handle_routes.js [hunk 1]
```diff
@@ -12,8 +12,8 @@ var myURIData = {};
 
     
 */
-function getCallbackStorage() {
-    return myURIData;
+function getCallbackStorage(URIData) {
```
Comment (perldork) [inline lib/handle_routes.js]: @alena0817 - this is a getter method - it should not take a parameter as that means it is not doing anything for us :).

File: lib/handle_routes.js [hunk 2]
```diff
@@ -100,6 +102,7 @@ RouteHandling.prototype = {
         } else {
             console.log("DEFAULT OPTION:");
             console.log(callbackStorage["default"](request, response));
+
             //return callBackStorage["default"](request, response);
```
Comment (perldork) [inline lib/handle_routes.js]: We shouldn't include commented out code in GIthub - that is what git is for, to remember what you did :)

File: lib/handle_routes.js [hunk 3]
```diff
@@ -109,27 +112,29 @@ RouteHandling.prototype = {
...
 
-Router = RouteHandling.create();
-
 
 myURIData[defaultKey] = readStaticFile;
+Router = RouteHandling.create(myURIData);
+
+
 
 //we keep it here because it applies to all apps.
```
Comment (perldork) [inline lib/handle_routes.js]: more commented out code.

File: lib/handle_routes.js [hunk 4]
```diff
@@ -109,27 +112,29 @@ RouteHandling.prototype = {
...
 Router.addRoute("/", ["GET"], function(request, response) {
     request.url = "index.html";
     readStaticFile(request, response);
 });
 /*
```
Comment (perldork) [inline lib/handle_routes.js]: a comment that looks like it was a hint about what you wanted to do, not explaining business logic?

File: lib/handle_routes.js [hunk 5]
```diff
@@ -109,27 +112,29 @@ RouteHandling.prototype = {
...
-move this part to the initial method that will be called before routing.addRoute
 
+this has to go into constructor.
 */
 
-
 module.exports = Router;
+//module.exports = new RouteHandling();
```
Comment (perldork) [inline lib/handle_routes.js]: commented out code

File: lib/handle_routes.js (hunk 1 above)
Comment (alena0817) [inline lib/handle_routes.js]: done

File: lib/handle_routes.js [hunk 6]
```diff
@@ -6,15 +6,6 @@ var mimeTypes =  require("./configuration.js").mimeTypes;
 var defaultKey = "default";
 var myURIData = {};
 
-/*
-    we want to have addRoute and evalutaeRoute functions to be inside an instance.
-    the other 2 functions can stay as is.
-
-    
-*/
-function getCallbackStorage() {
```
Comment (perldork) [inline lib/handle_routes.js]: Nice job removing unused code!

File: lib/handle_routes.js (hunk 3 above)
Comment (perldork) [inline lib/handle_routes.js]: Ah, I see why you didn't remove this one - it is business logic related 👍

**Pull request merged**